pkgs/applications/editors/emacs-24/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     sha256 = "1zflm6ac34s6v166p58ilxrxbxjm0q2wfc25f8y0mjml1lbr3qs7";
   };
 
-  patches = [ ./darwin-new-sections.patch ];
+  patches = [ ./darwin-new-sections.patch ./at-fdcwd.patch ];
```

Comment:
IMHO, `at-fdcwd.patch` should be activate only on Darwin to avoid side-effects (and re-builds) on other platforms.